Flow.Launcher.Core/Plugin/PluginInstaller.cs

```diff
@@ -71,8 +75,7 @@ internal static void Install(string path)
                         File.Create(Path.Combine(existingPlugin.Metadata.PluginDirectory, "NeedDelete.txt")).Close();
                     }
 
-                    UnZip(path, newPluginPath, true);
```

Comment:
this is not needed?

Reply:
The old code was unzipping the same file twice. New code unzips it once and then moves the directory in this step. You can test it, it works.

That said, perhaps the initial unzip should use a random directory inside the temp folder...

Reply:
oh good pick up, shouldnt unzip twice.

how does it handle unzipping same plugin to the same locations ?